content/en/security/code_security/iast/security_controls/_index.md

```diff
@@ -299,6 +299,62 @@ The following security control definition affects every `sql-sanitizer` package
 `SANITIZER:SQL_INJECTION:node_modules/sql-sanitizer/index.js:sanitize`
 
 
+{{% /collapse-content %}}
+
+{{% collapse-content title=".NET" level="h4" %}}
+
+## General syntax
+`TYPE:SECURE_MARKS:Assembly:Class:Method(ParameterTypes)[:ParameterIndexes]`
+
+<div class="alert alert-info">
+Parameter types must be fully qualified with their namespace. Example: `System.String`
```

Comment:
```suggestion
Parameter types must be fully qualified with their namespace. Example: <code>System.String</code><br /><br />
```
Some arcane formatting, brought to you by trial and error, to get the div to play a little more nicely 🙂